pytket_qir/generator.py

```diff
@@ -406,3 +405,107 @@ def condition_zero_block():
                     else:
                         get_gate(*qubits)
         return module
+
+    def command_to_module(
+        self, command: Command, circuit: Circuit, module: Module
```

Comment:
what is the module parameter used for?

Reply:
It is populated with the corresponding LLVM instructions for the `pytket` circuit. Then, it can generate QIR by calling the `.ir()` or `.bitcode()` methods.